files/pt-br/learn/common_questions/how_does_the_internet_work/index.md

```diff
@@ -0,0 +1,117 @@
+
+![Um roteador conectado a um modem](https://mdn.mozillademos.org/files/8451/internet-schema-6.png)
+
+Então nós estamos conectados à infraestrutuca telefônica. O próximo passo é enviar mensagens da nossa rede para a rede que nós desejamos alcançar. Para fazer isto, vamos precisar conectar nossa rede a um Provedor de Serviço de Internet (ISP, em inglês). Um ISP é uma compania que gerencia alguns roteadores especiais que são conectados e pode também acessar roteadores de outros ISPs. Então a mensagem da nossa rede é transportada para a rede de redes do ISP e então para a rede de destino. A Internet é composta por toda esta infraestrutura de redes.
+
+![Pilha completa da Internet](https://mdn.mozillademos.org/files/8453/internet-schema-7.png)
+
+### Encontrando computadores
+
+Se você quer enviar uma mensagem para um computador, você precisa especificar qual é este computador. Por isso, qualquer computador conectado à uma rede possui um único endereço de identificação, chamado de "Endereço IP" (onde IP, do inglês _Internet Protocol_, significa Protocolo de Internet). Este é um endereço composto por uma série de 4 números separados por pontos, por exemplo: `192.168.2.10`.
+
+Isto é perfeito para computadores, mas nós seres humanos temos dificuldades para lembrar estes endereços. Para tornar as coisas mais fáceis, nós podemos dar apelidos aos endereços IP que nós humanos podemos compreender, chamados _nome de domínio_. Por exemplo, `google.com` é um nome de domínio usado para "apelidar" o endereço `142.250.190.78` (no momento em que este artigo foi escrito. Endereços de IP podem mudar). Então, usando o nome de domínio é uma forma mais simples de encontrar um computador na Internet.
+
+![Mostrando como um nome de domínio pode criar um apelido para um endereço IP](https://mdn.mozillademos.org/files/8405/dns-ip.png)
+
+### A Internet e a Web
+
+Como você deve ter notado, quando navegamos na Web com nossos navegadores, normalmente utilizamos os nomes de domínios para chegar a um website. Isto significa que a Internet e a Web são a mesma coisa? Não tão simples assim. Como vimos, a Internet é uma infraestrutura técnica que permite conectar bilhões de computadores. Entre estes computadores, alguns computadores (chamados de _servidores Web_) podem enviar mensagens intelegíveis para navegadores Web. A _Internet_ é a infraestrutura, enquanto a _Web_ é um serviço construído sob esta infraestrutura. Vale a pena notar que existem diversos outros serviços que funcionam na Internet, tais como email e {{Glossary("IRC")}}.
```

Comment:
```suggestion
Como você deve ter notado, quando navegamos na Web com nossos navegadores, normalmente utilizamos os nomes de domínios para chegar a um website. Isto significa que a Internet e a Web são a mesma coisa? Não é tão simples assim. Como vimos, a Internet é uma infraestrutura técnica que permite conectar bilhões de computadores. Entre estes computadores, alguns computadores (chamados de _servidores Web_) podem enviar mensagens inteligíveis para navegadores Web. A _Internet_ é a infraestrutura, enquanto a _Web_ é um serviço construído sob esta infraestrutura. Vale a pena notar que existem diversos outros serviços que funcionam na Internet, tais como email e {{Glossary("IRC")}}.
```

Reply:
Corrigindo 2 erros de digitação, estava faltando o "é" de "It's not that simple", a palavra "intelegíveis" é com i, assim "inteligíveis".